pepdbagent/modules/annotation.py

```diff
@@ -463,3 +481,82 @@ def get_project_number_in_namespace(
             return result[0]
         except IndexError:
             return 0
+
+    def get_by_rp_list(
+        self,
+        registry_paths: List[str],
+        admin: Union[str, List[str]] = None,
+    ) -> AnnotationList:
+        """
+        Get project annotations by providing list of registry paths.
+
+        :param registry_paths: registry path string or list of registry paths
+        :param admin: list of namespaces where user is admin
+        :return: pydantic model: AnnotationReturnModel(
+            limit:
+            offset:
+            count:
+            result: List [AnnotationModel])
+        """
+        admin_tuple = tuple_converter(admin)
+
+        if isinstance(registry_paths, list):
+            or_statement_list = []
+            for path in registry_paths:
+                try:
+                    namespace, name, tag = registry_path_converter(path)
+                    or_statement_list.append(
+                        and_(
+                            Projects.name == name,
+                            Projects.namespace == namespace,
+                            Projects.tag == tag,
+                            or_(
+                                Projects.namespace.in_(admin_tuple),
+                                Projects.private.is_(False),
+                            ),
+                        )
+                    )
+                except RegistryPathError as err:
+                    _LOGGER.error(str(err), registry_paths)
+                    continue
```

Comment:
Is it a good idea to ignore registry paths that are invalid? Trying to think of a scenario where this bites us.

Reply:
Solved this issue by returning None instead of skipping error